.github/workflows/changelog.yml

```
@@ -32,7 +32,8 @@ jobs:
     steps:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
         with:
-          fetch-depth: 0
+          # Fetch complete history depth only if the PR is not a chore.
+          fetch-depth: ${{ !contains(github.event.pull_request.labels.*.name, 'dependencies') && !contains(github.event.pull_request.labels.*.name, 'Skip Changelog') && !contains(github.event.pull_request.title, '[chore]') && '0' || '1' }}
```

I got used to leverage `fromJSON` for such cases, __for me__ it is more readable, (warning: comment compiled):

```suggestion
          fetch-depth: ${{ fromJSON('["0", "1"]')[contains(github.event.pull_request.title, '[chore]') || contains(github.event.pull_request.labels.*.name, 'Skip Changelog') || contains(github.event.pull_request.labels.*.name, 'dependencies')] }}
```